html-forms/index.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>HTML Forms</title>
+</head>
+
+<body>
+  <h1>Coding Form!</h1>
+
+  <div>
+    <form action="" method="get" class="form-example">
+      <div class="form-example">
+        <label for="email">Email</label>
+        <div>
+          <input type="text" email="email" id="email" required>
+        </div>
+      </div>
+    </form>
+  </div>
+
+  <div>
+    <label for="why-cool">Why is coding cool?</label>
+  </div>
+  <div>
+    <form action="" method="get" class="form-example">
+      <div class="form-example">
+        <select name="options" id="why-cool">
+          <option value="first">Select an option</option>
+          <option value="second">Because it's interesting</option>
+          <option value="third">Because it's fun</option>
+          <option value="fourth">Because it's powerful</option>
+        </select>
+      </div>
+    </form>
+  </div>
+
+  <div>
+    <label for="fav-language">Favorite Language (choose one)</label>
+  </div>
+  <div>
+    <form action="" method="get" class="form-example">
```

Comment:
All of the inputs in this form should be held in one `form` element, not an individual `form` element for each `input`.